group/templates/group/groupList.html

```diff
@@ -0,0 +1,141 @@
+  <!-- Magnific Popup core JS & CSS -->
+  <link rel="stylesheet" href="/css/magnific-popup.css">
+  <link rel="stylesheet" href="/css/mfp-fade.css">
+  <script src="/js/jquery.magnific-popup.js"></script>
+  <script src="/js/popup-setting.js"></script>
+
+</head>
+{% extends "index/base.html" %} 
+{% load static %} 
+{% block body_block %}
+
+<script language="javascript" type="text/javascript" 
+        src="//cdnjs.cloudflare.com/ajax/libs/flot/0.8.2/jquery.flot.min.js"></script>
+<script language="javascript" type="text/javascript" 
+        src="//cdnjs.cloudflare.com/ajax/libs/flot/0.8.2/jquery.flot.pie.min.js"></script>
+<div class="container-fluid">
+  <div class="row-fluid">
+    <div class="span12">
+      <div class="form-group">
+        <p><a class="btn" href="#" style="float:right">Create &raquo;</a></p>
+          <input class="form-control input-sm" type="text" id="inputSmall" placeholder="Group Name"
+                 style="float:right">
+      </div>
+    </div>
+  </div>
+</div>
+<hr> 
+{% block list_block %}
+<div class="contest">
+  <h2 style="border-bottom:1px solid; padding:2px; float:left; font-family:sans-serif">
+    Group List
+  </h2>
+  <br>
+  <br>
+  <br>
+  <br>
+  <div class="filter" style="float:right">
+    <div class="btn-group" data-toggle="buttons-radio" style="float:right">
+      <button type="button" class="btn btn-primary">My Group</button>
+      <t>
+        <button type="button" class="btn btn-primary">All Group</button>
+    </div>
+  </div>
+  <a href="" class="btn btn-sm btn-default" data-toggle="modal" 
+     data-target="#searchGroupModal" style="float:left">Search group</a>
+  <table class="table table-hover">
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>Owner/TA</th>
+        <th>Member Number</th>
+      </tr>
+    </thead>
+    <tbody>
+      {% for group in G_data %}
+      <tr>
+        <td>
+          <a href="{% url 'detail' group.gid %}">{{ group.name }}</a>
+        </td>
+        <td>{{ group.ta_name }}</td>
+        <td>{{ group.mem_number }}</td>
+        <td>
+          <p><a class="btn" href="#" style="float:right">Delete &raquo;</a>
+        </td>
+      </tr>
+      {% endfor %}
+    </tbody>
+  </table>
+  <ul class="pager">
+    <li class="previous disabled"><a href="#" style="color:gray">← Previous</a>
+    </li>
+    <li class="next"><a href="#" style="color:black">Next →</a>
+    </li>
+  </ul>
+</div>
+{% endblock list_block %}
+<hr>
+
+<!---Modal Section-->
+<!--1. Modal for search group-->
+<div class="modal fade" id="searchGroupModal" tabindex="-1" role="dialog" 
+     aria-labelledby="myModalLabel" aria-hidden="true">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="btn-default" class="close" data-dismiss="modal" aria-hidden="true">
+          &times; 
+        </button>
+        <h4 class="modal-title" id="myModalLabel">Search</h4>
```

Comment:
Any more meaningful id of this text?
I prefer each id of html has its own meaning.